ui/src/components/ConfirmReleaseIP.vue

```diff
@@ -0,0 +1,38 @@
+<!--
+  Copyright (C) 2023 Nethesis S.r.l.
+  SPDX-License-Identifier: GPL-3.0-or-later
+-->
+<template>
+  <NsModal
+    size="default"
+    :visible="isShown"
+    @modal-hidden="$emit('hide')"
+    @primary-click="$emit('confirm')"
+  >
+    <template slot="title"
+      >{{ $t("unban.unban_ip") }}: {{ ban.value }}</template
+    >
+    <template slot="content">
+      <div>{{ $t("unban.confirm_unban_ip_message") }}</div>
+    </template>
+    <template slot="secondary-button">{{ core.$t("common.cancel") }}</template>
+    <template slot="primary-button">{{ core.$t("common.delete") }}</template>
```

Comment:
```suggestion
    <template slot="primary-button">{{ core.$t("unban.delete") }}</template>
```

Reply:
are you sure for this translation ?

Reply:
Why not? The user just clicked "Unban" on the kebab menu that shows this modal, it is a good a practice to repeat the same verb on the confirmation button